content/intermediate/operations/stack-policy-and-prevention-controls/index.md

```diff
@@ -125,14 +199,26 @@ Resources:
     Properties:
       TopicName: Topic-2
 ```
+   :::::tabs{variant="container"}
+	::::tab{id="cloud9" label="Cloud9"}
+  Run the following command in the terminal to **Create Stack**:
+  :::code{language=shell showLineNumbers=false showCopyAction=true}
+  aws cloudformation create-stack \
+      --stack-name cfn-workshop-stack-policy-deletion \
+      --template-body file://deletion-policy-lab.yaml
+  :::
+  ::::
+
+	::::tab{id="local" label="Local development"} 
 
 1. Navigate to the [AWS CloudFormation Console](https://console.aws.amazon.com/cloudformation/).
 2. From **Create stack**, choose **With new resources (standard)**.
 3. From **Specify template**, choose **Upload a template file**. Upload the `deletion-policy-lab.yaml` template, and choose **Next**.
-4. Enter a Stack name. For example, specify `deletion-policy-lab`. Choose **Next**.
+4. Enter a Stack name. For example, specify `cfn-workshop-stack-policy-deletion`. Choose **Next**.
 5. Choose to accept default values on the **Configure stack options page**; scroll to the bottom of the page, and choose **Next**.
-6. In the next page, choose **Create stack**.
-
+6. In the next page, choose **Submit**.
+   ::::
+   :::::
```

Comment:
Please fix stack name on line 227 'deletion-policy-lab' to
```
2. Select the stack named `cfn-workshop-stack-policy-deletion`, and choose **Delete**. Next, choose **Delete** to confirm.
```

Reply:
updated